8.frontend-react/crm-frontend/src/components/table/Table.jsx

```diff
@@ -5,6 +5,7 @@ import './table.scss'
 function Table(props) {
 
     const {columns, data} = props;
+    console.log("data:", data);
```

Comment:
remove console logs after you done with them 
you don't want to release code having logs of all your data and stuff 👻